lib/ssl/src/dtls_handshake.hrl

```diff
@@ -51,7 +51,6 @@
 %% RFC 7764  Datagram Transport Layer Security (DTLS) Extension to Establish Keys
```

Comment:
RFC7764 seems not correct any longer. Replace it with RFC5764.

Reply:
Yep, seems like an old typo. Fixed.